x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/constants.ts

```diff
@@ -118,3 +118,42 @@ export const timeUnits = [
     }),
   },
 ];
+
+/*
+ * Labels for fixed intervals
+ */
+export const fixedIntervalUnits = [
```

Comment:
I see we already have a `timeUnits` const. Just want to confirm these values should be separate?

Reply:
micros and nanos are technically not supported for the fixed interval. 
btw, I saw this: https://github.com/elastic/kibana/issues/137081

I think we can unite these consts if we remove everything smaller then a minute